pkg/propertychecker/azure/checker.go

```diff
@@ -0,0 +1,179 @@
+//
+//Copyright (c) Microsoft Corporation.
+//Licensed under the MIT license.
+
+// Package azure provides property checkers for Azure-specific cluster requirements.
+// It checks whether the cluster can meet the requirement defined by the property selector.
+package azure
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"go.goms.io/fleet/pkg/utils"
+	"go.goms.io/fleet/pkg/utils/labels"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/klog/v2"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	computev1 "go.goms.io/fleet/apis/protos/azure/compute/v1"
+	"go.goms.io/fleet/pkg/clients/azure/compute"
+	"go.goms.io/fleet/pkg/propertyprovider/azure"
+)
+
+const (
+	// MaxVMInstanceCapacity defines the maximum allowed VM instance capacity for SKU capacity requirements.
+	// This limit is set to prevent excessive resource requests and potential quota issues.
+	// The value is constrained by uint32 maximum (4,294,967,295) but set to a reasonable upper bound
+	// of 200 for most production workloads.
+	MaxVMInstanceCapacity = 200
```

Comment:
```suggestion
	maxVMInstanceCapacity = 200
```

Why do you make it public when it's only used internally.